### DISCUSSION TITLE
Is there any way to prevent Pyinstaller from adding libpython3.7m.so.1.0 ?

### DISCUSSION THREAD

**Discussion by ziXet** [pyinstaller/pyinstaller#6628] [Help]
Hi,

I want to use the installed python of the host machine to reduce the size of the generated artifact. (Linux)

Thanks

**Comment by rokm** [comment]
There is not. On the other hand, nothing prevents you from modifying PyInstaller sources to achieve what you want, either.

**Comment by bwoodsend** [comment]
If you want to use the host Python then what on earth do you need PyInstaller for?

**Reply by ziXet** [reply]
> If you want to use the host Python then what on earth do you need PyInstaller for?

haha. It's complicated.

Thanks @rokm . I got my answer.
